src/tickets.rs

```diff
@@ -340,4 +342,28 @@ mod tests {
             )
         });
     }
+
+    #[test]
+    fn test_space_efficient_shuffler_skewed_tickets() {
+        let mut rng = rand::rngs::mock::StepRng::new(1, 0);
+        const MAX_PRIZE_COUNT: usize = 10;
+        const NUM_USERS: usize = 11;
+        let prizes =
+            Vec::from_iter((0..MAX_PRIZE_COUNT).map(|x| PrizeBuilder::new().count(x).name(format!("{x}")).build()));
+        // set the first user would always not have the prize
+        let (mut users, log) = (0..NUM_USERS)
+            .map(|u| CapacityOneUser::new(if u == 0 { 1 } else { 1000 }))
+            .collect::<(Vec<_>, Vec<_>)>();
+        let mut ses = SpaceEfficientShuffler::new(&mut users);
+        let mut prizes = prizes.iter().peekable();
+        while ses.try_draw_one(&mut rng, &mut prizes) {}
+        assert_eq!(
```

Comment:
Since here the test is using `StepRng`, maybe the assertion could be more explicit: we _know_ the answers.